README.md

```diff
@@ -11,4 +11,34 @@ the folder 3rdparty.
 
 Screenshots
 -----------
-![Alan Wakes American Nightmare 1](screenshots/awan1.png)
\ No newline at end of file
+![Alan Wakes American Nightmare 1](screenshots/awan1.png)
```

Comment:
I'll address the other parts later, but in this case it's not really possible for me to not change this line. The reason it gets changed is because there's no newline at the end of it without this pull request, and I can't really make a new line after this without adding a newline here.

Reply:
Ok, I understand, if that cannot be changed then leave it that way